package com.gustavo.auth.exception.exceptions;

public class EventNotFoundException extends RuntimeException {

    public EventNotFoundException() { super("Evento n√£o encontrado!"); }

    public EventNotFoundException(String message) { super(message); }
}
